### DISCUSSION TITLE
Would like an example of injecting into Electron

### DISCUSSION THREAD

**Discussion by suiniara** [ShellWen/v8_killer#18] [General]
Hello, excuse me, would like an example of injecting into Electron? I tried injecting and got the following error:
```
[-] v8_script_compiler_compile_function_internal not found
```
I'm not sure if I did something wrong.
Thank you for your reply.

**Comment by ShellWen** [comment]
It's seems that Windows version of Electron remove that symbol. We have no idea on that.

**Reply by suiniara** [reply]
Thank you for your response. In that case, should I consider using Linux or macOS to solve this problem?

**Comment by ShellWen** [comment]
Not sure. You can test it.
